webplugin/plugin.js

```diff
@@ -62,6 +62,14 @@ var kmCustomIframe =
         'margin:0;'+
         'height:100vh;'+
         'width:100vw;'+
+    '} \n' + 
+    '.kommunicate-custom-iframe.chat-popup-widget-horizontal { ' + 
+    '   width: 390px;' + 
+    '   height: 80px;' + 
+    '} \n' + 
+    '.kommunicate-custom-iframe.chat-popup-widget-vertical { ' + 
+    '   width: 390px;' + 

```

Comment:
@CaptainT33mo  what about devices with width less than 390px?

Reply:
@akshatdhabi will add a media query for smaller screen devices.